changelog/next/features/3114--zeek-tsv-format.md

```diff
@@ -0,0 +1,2 @@
+The `zeek-tsv` format enables the user to parse and print Zeek TSV logs inside
+a pipeline.
```

Comment:
```suggestion
The `zeek-tsv` format parses and prints Zeek's native tab-separated value (TSV) 
representation of logs.
```

Reply:
Fixed in https://github.com/tenzir/vast/pull/3114/commits/d01e63a216f6cf0cb80a82adb3a4ae4e89a9286c